mmpose/models/mesh_heads/discriminator.py

```diff
@@ -0,0 +1,289 @@
+# ------------------------------------------------------------------------------
+# Adapted from https://github.com/akanazawa/hmr
+# Original licence: Copyright (c) 2018 akanazawa, under the MIT License.
+# ------------------------------------------------------------------------------
+
+from abc import abstractmethod
+
+import torch
+import torch.nn as nn
+
+from .geometric_layers import batch_rodrigues
+
+
+class LinearModel(nn.Module):
```

Comment:
the name `LinearModel` is too generic.
how about replace it with a more specific name

Reply:
OK, I will use` BaseDiscriminator`.